### PR TITLE
Build and dev containers for CentOS Stream 8.

### PR DESCRIPTION
**Pull request by aressem**
I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

The PR for the dev image will fail because it depends on published build image.

### CODE REVIEW

File: dev/centos-stream8/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,43 @@
+# Copyright Yahoo. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+FROM docker.io/vespaengine/vespa-build-centos-stream8:latest
+
+ENV container docker
+
+# Fixes a problem with Rpmdb checksum cache conflicts: https://github.com/CentOS/sig-cloud-instance-images/issues/15
+RUN dnf -y install dnf-plugin-ovl
```
Comment (toregge) [inline dev/centos-stream8/Dockerfile:7]: Consider installing `dnf-plugin-ovl` earlier, i.e. in build image.

File: dev/centos-stream8/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+        nmap-ncat \
+        vim \
+        emacs \
+        man-db \
+        man-pages \
+        hunspell-en && \
+    dnf -y install \
+        gcc-toolset-11-annobin-plugin-gcc \
+        gcc-toolset-11-gdb \
+        gcc-toolset-11-libasan-devel && \
```
Comment (toregge) [inline dev/centos-stream8/Dockerfile:33]: Consider adding `gcc-toolset-11-libtsan-devel` too.

**Review by toregge — APPROVED** [review]

File: dev/centos-stream8/Dockerfile (hunk 1 above)
Comment (aressem) [reply dev/centos-stream8/Dockerfile]: Fixed.

File: dev/centos-stream8/Dockerfile (hunk 2 above)
Comment (aressem) [reply dev/centos-stream8/Dockerfile]: Fixed

**Review by toregge — APPROVED** [review]

**Pull request merged**